content/exchange/artifacts/Linux.Forensics.EnvironmentVariables.yaml

```diff
@@ -0,0 +1,76 @@
+name: Linux.Forensics.EnvironmentVariables
+author: Idan Beit-Yosef @ ibyf0r3ns1cs
+description: |
+  This artifact detects potential persistence mechanisms on Linux systems by analyzing environment variable files and login scripts.  
+  Adversaries may modify these files to establish persistence by injecting commands that execute malicious scripts or binaries upon user login or shell initialization.  
+  This artifact focuses on identifying such modifications by scanning for environment variable alterations, network utilities, and scripting languages commonly used in attacks.  
+  The globs for file paths can be added using a comma-separated list, and the filters are regex patterns that can be modified as needed to tailor the artifact's detection capabilities.
+  
+  **MITRE ATT&CK**: [T1546.004](https://attack.mitre.org/techniques/T1546/004/)
+
+reference:
+  - https://unit42.paloaltonetworks.com/unit42-new-iotlinux-malware-targets-dvrs-forms-botnet/
+  - https://intezer.com/blog/research/kaiji-new-chinese-linux-malware-turning-to-golang/
+  - https://www.anomali.com/blog/illicit-cryptomining-threat-actor-rocke-changes-tactics-now-more-difficult-to-detect
+  - https://www.anomali.com/blog/pulling-linux-rabbit-rabbot-malware-out-of-a-hat
+  - https://blog.sucuri.net/2018/05/shell-logins-as-a-magento-reinfection-vector.html
+
+parameters:
+  - name: LinuxEnvGlobs
+    default: /home/*/.bashrc,/home/*/.bash_profile,/home/*/.bash_login,/home/*/.profile,/home/*/.zshrc,/etc/profile,/etc/environment,/home/*/.bash_logout
```

Comment:
This is better as a CVS parameter type then you dont need to split it - see how the FileFinder does it